### DISCUSSION TITLE
SUPPORT FOR AZURE OPENAI

### DISCUSSION THREAD

**Discussion by hemantrao12** [TEN-framework/ten-framework#538] [General]
Team, i have azure open ai and other service keys, how do i use the same for my LLM work

**Comment by plutoless** [comment]
it's already supported.
for non-realtime use openai_chatgpt_python, set following properteis,
```
"azure_endpoint"
"azure_api_version"
"vendor": "azure"
```


for realtime use openai_v2v, set following properties,

```
"vendor": "azure"
"path"
"base_uri"
```
